test/serialize.js

```diff
@@ -133,14 +207,47 @@ describe('cookie.serialize(name, value, options)', function () {
 
   describe('with "path" option', function () {
     it('should serialize path', function () {
-      assert.equal(cookie.serialize('foo', 'bar', { path: '/' }), 'foo=bar; Path=/')
-    })
+      const validPaths = [
+        '/',
+        '/login',
+        '/foo.bar/baz',
+        '/foo-bar',
+        '/foo=bar?baz',
+        '/foo"bar"',
+        '/../foo/bar',
+        '../foo/',
+        './'
+      ];
+
+      validPaths.forEach((path) => {
+        assert.equal(
+          cookie.serialize('foo', 'bar', { path }),
+          `foo=bar; Path=${path}`,
+          `Expected serialized value for path: "${path}"`
+        );
+      });
+    });
 
     it('should throw for invalid value', function () {
-      assert.throws(cookie.serialize.bind(cookie, 'foo', 'bar', { path: '/\n' }),
-        /option path is invalid/)
-    })
-  })
+      const invalidPaths = [
+        '/\n',
+        '/foo\u0000',
+        '/foo bar',
```

Comment:
This wasn't invalid, should it be? The spec says `<any CHAR except CTLs or ";">` so I believe space is fine?

Reply:
You are correct. Space should be acceptable.